packages/terra-dropdown-button/src/Item.jsx

```diff
@@ -44,16 +44,18 @@ const Item = ({
     navigate outside the dropdown with the dropdown still open if nothing else is pressed after opening the menu.
   */
   return (
-    <li tabIndex="-1" role="menuitem">
+    <li
+      {...customProps}
+      onClick={(event) => {
+        requestClose(() => {
+          onSelect(event, metaData);
+        });
+      }}
+      tabIndex="0"
+      role="menuitem"
+    >
       <div
-        {...customProps}
-        onClick={(event) => {
-          requestClose(() => {
-            onSelect(event, metaData);
-          });
-        }}
-        role="menuitem"
-        tabIndex="0"
+        role="none"
```

Comment:
just curious, what happens if we don't have role="none", does the screenreader read out unwanted information from this div?

Reply:
On Mac VoiceOver it still reads how we want it to which is "{content} menu item", I don't have a good way to test JAWS or VoiceOver on IOS.

The concern here is that `div` has no implicit ARIA semantics or role associated to it so to make absolutely sure it doesn't get read as `Group` or something else by any screenreader I have `role="none"`